bin/setup

```diff
@@ -52,7 +52,7 @@ FileUtils.chdir APP_ROOT do
   puts "\n== Removing old logs and tempfiles =="
   system! 'bin/rails log:clear tmp:clear'
 
-  puts "\n== Restarting application server =="
-  # system! 'bin/rails restart'
-  system! 'bin/rails server -p 8080'
+  # puts "\n== Restarting application server =="
+  # # system! 'bin/rails restart'
+  # system! 'bin/rails server -p 8080'
```

Comment:
I have commented this part to reuse it in Dockerfile

Reply:
What do you mean by "to re-use in Dockerfile"?

I don't think we should comment this out as it'll have implications to our 1 click Digital Ocean deploy https://github.com/dradis/dradis-ce/blob/develop/.do/deploy.template.yaml#L16

What about adding another bin file but for Docker? See: https://github.com/dradis/dradis-ce/blob/develop/bin/heroku